lua/telescope/utils.lua

```diff
@@ -22,6 +22,9 @@ end
 utils.get_default = function(x, default)
   return utils.if_nil(x, default, x)
 end
+utils.cycle = function(i, n)
```

Comment:
empty line between functions

Reply:
Second time, I'm close to raising an issue on stylua :laughing:

Reply:
stylua never deleted a line for me :thinking:

Reply:
I meant that I want stylua to add a line back like black for Python would to fix my twitching fingers